### DISCUSSION TITLE
Internal Link Issues

### DISCUSSION THREAD

**Discussion by maybenikhil** [squidfunk/mkdocs-material#3410] [Q&A] [answered]
Hello! I am writing some documentation for a radio link and when i try to use internal links I have a lot of issues. The current method i am using is quite clunky, because the default way that relative links SHOULD work is not working. For example on one page `site/quick-start/tx-prep/` I try to link to some other page `site/hardware/inverter-mod`. The method I would think would work given all other mkdocs sites i have seen would be on the first page to link as `[inverter mod](hardware/inverter-mod.md)`. However, when I build this on my local serve this links to `site/quick-start/tx-prep/hardware/inverter-mod.md` which is obviously not the correct expectation. Is there anyone else who have had this issue? The workaround I have had to use is to link as follows: `[inverter mod](../../hardware/inverter-mod.md)` which gets incredibly complex as the directories get deeper. Thanks all!

**Comment by squidfunk** [comment, marked as answer]
MkDocs only supports relative links. However, there are some plugins that implement additional linking capabilities. It's a good idea to check the [MkDocs plugin wiki](https://github.com/mkdocs/mkdocs/wiki/MkDocs-Plugins#links--references).
